core/src/blocktree.rs

```diff
@@ -118,13 +120,108 @@ impl SlotMeta {
     }
 }
 
+#[cfg(feature = "erasure")]
+#[derive(Clone, Debug, Default, Deserialize, Serialize, Eq, PartialEq)]
+/// Erasure coding information
+pub struct ErasureMeta {
+    pub slot: u64,
```

Comment:
is this also the Meta's key in the column?  need to be included in the data?

Reply:
It will be removed, that's an artifact of how I initially built ErasureMeta